### DISCUSSION TITLE
Adding Android Go testing feature

### DISCUSSION THREAD

**Discussion by EmilioAVazquez** [GoogleChrome/lighthouse#11750] [Ideas]
Hi there,
I'm planning to add new functionality to Lighthouse to account for efficiency on Android Go devices. Adroid Go devices are usually low in power and memory. Thus, having Lighthouse to test for this constrains is essential.

If you think this is already integrated, or if it has already been done, please let me know! 

Also if you want to help out, just send me an email at evazqu28@uic.edu !

Emilio

**Comment by patrickhulce** [comment]
Neat idea @EmilioAVazquez! you might also be interested in https://github.com/GoogleChrome/lighthouse/issues/5057 which explored some concepts here and of course checkout the [DevTools memory APIs](https://chromedevtools.github.io/devtools-protocol/tot/Memory/) :)
